preprocess.py

```diff
@@ -16,7 +16,13 @@
 
 def overlap(sightline, data:DesiMock, id):
     '''
-    here sightline = data.get_sightline(id=id)
+    Deal with the overlapping area between different cameras so that the result will not contain the overlaps.
```

Comment:
- please move `get_split_point` and `get_between` outside the `overlap` function
- documents are required for `get_split_point` and `get_between` function
- have you ever visually tested for a large number of mock spectra that using the average points to split the two camera is correct, i.e., no wired features?